hw4/js/to-do.js

```diff
@@ -0,0 +1,52 @@
+//add the input to the to-do list when clicked
+$(document).ready(
+    $("#new-item").on('click', function() {
+        var user_input = $("#item-input").val();
+        $("#to-do-list").prepend("<div> <button class='to-do-item'>"+ "Mark" + "</button> " + user_input + "</div>");
+    })
+);
+
+//when the button of to-do list item take actions 1)move it to complete list, 2) changestyle, 3) change button content
+$("#to-do-list").on('click', 'div', function() {
+  var input_content = $(this);
+  $("#completed-list").prepend(input_content);
+  $("#completed-list > div > button").html("Unmark")
+  $("#completed-list > div > button").removeClass("to-do-item");
+  $("#completed-list > div > button").addClass("completed-item");
+});
+
+//when the button of complete list item take actions 1)move it to to-do list, 2) changestyle, 3) change button content
+$("#completed-list").on('click', 'div', function() {
+  var transfer_content = $(this);
+  $("#to-do-list").prepend(transfer_content);
+  $("#to-do-list > div > button").html("Mark Complete")
```

Comment:
Originally in when you add an item to the todo, it says "mark". When an item is transferred from the completed list, it says "mark completed". A little inconsistent.